robot/markers.py

```diff
@@ -168,9 +168,12 @@ def spherical(self) -> SphericalCoord:
         """
         return SphericalCoord(*self._raw_data['spherical'])
 
+    def __repr__(self):
+        return "<robot.markers.{} >".format(str(self))
+
     def __str__(self):
         bearing = self.spherical.rot_y_degrees
-        return "<Marker {}: {:.0f}° {}, {:.2f}m away>".format(
+        return "Marker {}: {:.0f}° {}, {:.2f}m away".format(
             self.id,
             abs(bearing),
             "right" if bearing > 0 else "left",
```

Comment:
Do we need to specify "left" and "right" here if this is a bearing?

Also, I think clockwise and anti-clockwise might be more appropriate here as they are less ambiguous.

Reply:
I think the issue here is that 'bearing' is the wrong variable name. I don't like mentioning clockwise or counter clockwise, as it's given without context here, and they might think it's the orientation of the marker

Reply:
either way, this isn't changed in this PR and thus should be fixed separately

Reply:
I think `bearing` is probably fine here, if not 100% precise. The variable holds the bearing of the marker relative to the bearing of the camera. I feel using "left/right" to indicate "to the left/right of the camera" is acceptable here.